src/Sql/Sql/Elastic Jobs/Cmdlet/JobStep/AddAzureSqlElasticJobStep.cs

```diff
@@ -225,44 +225,44 @@ public class AddAzureSqlElasticJobStep : AzureSqlElasticJobStepCmdletBase<AzureS
         /// Gets or sets the credential name
         /// </summary>
         [Parameter(
-            Mandatory = true,
+            Mandatory = false,
```

Comment:
can you reflect this syntax change in changelog?

Reply:
done